x11-misc/yagostatus/yagostatus-0.2.0.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit golang-build
+
+EGO_PN="github.com/burik666/yagostatus"
+DESCRIPTION="Yet Another i3status replacement written in Go"
+HOMEPAGE="https://github.com/burik666/yagostatus"
+LICENSE="GPL-3"
+SLOT="0/${PVR}"
+IUSE="pie"
+
+EGO_VENDOR=(
+	"golang.org/x/net 32a936f github.com/golang/net"
+	"gopkg.in/yaml.v2 5420a8b github.com/go-yaml/yaml"
+)
+
+QA_PRESTRIPPED="usr/bin/yagostatus"
```

Comment:
it's not needed if you omit passing those ldflags below and let portage strip.
letting portage strip is the preferred way.